Lib/profile/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .profile import run
+from .profile import runctx
+from .profile import Profile
+from .profile import _Utils
+
+__all__ = ['run', 'runctx', 'Profile']
```

Comment:
Nit: note that this isn't exactly enough to satisfy "backwards compatibility". Users that monkey-patched `profile.runctx` will now still be patching it but it won't be very effective.